Pyrado/pyrado/environments/mujoco/quanser_qube.py

```diff
@@ -143,7 +136,7 @@ def _mujoco_step(self, act: np.ndarray) -> dict:
         return dict(
             qpos=qpos,
             qvel=qvel,
-            failed=mjsim_crashed or state_oob,
```

Comment:
why can we delete mjsim_crashed?

Reply:
Fixed it in aeb632f73a800f064eca9a6e9a19d71d57bcb75e